pkg/cmd/kubectlenv/kubectlenv.go

```diff
@@ -32,18 +32,30 @@ func NewCmdKubectlEnv(f util.Factory, ioStreams util.IOStreams) *cobra.Command {
 See each sub-command's help for details on how to use the generated script.
 
 The generated script points the KUBECONFIG environment variable to the currently targeted shoot, seed or garden cluster.
+To point the KUBECONFIG to the targeted cluster in all shell sessions consider adding the script to your shell's startup configuration.
```

Comment:
To remove duplication, we could rephrase the long description to the following
```
Generate a script that points KUBECONFIG to the currently targeted shoot, seed, or garden cluster for the specified shell. To apply this setting automatically in every shell session, consider adding the generated script to your shell's startup configuration.

See each sub-command's help for details on how to use the generated script.
```

Reply:
Fixed via [`64696f7` (#490)](https://github.com/gardener/gardenctl-v2/pull/490/commits/64696f7df1ff0057b7a4e821a76a65987932acce)